drivers/postgres/postgres_test.go

```diff
@@ -51,6 +49,11 @@ func (suite *PostgresTestSuite) BeforeTest(_, _ string) {
 }
 
 func (suite *PostgresTestSuite) AfterTest(_, _ string) {
+	if pg, ok := suite.driver.(*postgres); ok && pg != nil {
+		err := pg.db.Close()
```

Comment:
We already have the db instance in `suite.db`. Do we need to go via the driver interface?

Reply:
I am going to revamp this, the suite shouldn't have the driver.